search/includes/classes/class-versioning.php

```diff
@@ -14,6 +14,8 @@ class Versioning {
 	const INDEX_VERSIONS_SELF_HEAL_LOCK_CACHE_GROUP          = 'vip_search';
 	const INDEX_VERSIONS_SELF_HEAL_LOCK_CACHE_TTL            = 10;
 	const INDEX_VERSIONS_SELF_HEAL_LOCK_CACHE_TTL_ON_FAILURE = 60 * 10; // 10 minutes
+	const VERSIONING_JOB_DEFAULT_PRIORITY                    = 15;
```

Comment:
I think adding the `INACTIVE` somewhere in the name makes it clearer. What do you think?
```suggestion
	const INACTIVE_VERSION_JOB_DEFAULT_PRIORITY                    = 15;
```

Reply:
Fixed in 09af12f04